### ISSUE TITLE
[Feature Request] Restore playback state on startup

### ISSUE THREAD

**Issue by waddyhussain** [Taiko2k/TauonMusicBox#757]
Option in settings to enable 'restore playback state on startup', saves playback position and play/pause state when closing program so can be restored when program is reopened.

**Comment by waddyhussain** [comment]
Found the 'resume-playback-on-restart' option on the config file, this works if you close the player while music is playing, but if playback is paused when you close it then the song starts again from the beginning if you reopen and press play

**Comment by Taiko2k** [comment]
Yes, it's not a super simple fix because Tauon currently doesn't have a function that allows it to go directly from stopped state to paused state, so implementing the paused aspect will take a little extra work.

**Comment by speakn2pabst** [comment]
In the situation described, restarting the program from a paused state within a track, is it really necessary to return to the paused state?  I think it's sufficient to be in the 'stopped' state on restart but when 'play' is started beginning at the 'paused' location.  This would be functionally the same to the user.

This comment was offered from a logic perspective without understanding your implementation from a code perspective.

**Comment by Taiko2k** [comment]
I think it would make sense to be paused, so the seek bar will show where it will resume from. Remember stop clears all info of the now playing track.

As for the code, currently there is no internal programmatic way to resume directly to the paused state, which makes this feature request little inconvenient to implement.
